lib/modules/filteReddit.js

```diff
@@ -37,6 +37,7 @@ module.options = {
 	// valid types are: text, boolean (if boolean, value must be true or false)
 	// for example:
 	NSFWfilter: {
+		title: 'NSFW Filter',
```

Comment:
I got @dhensche to sneak it in for me https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/3537 :grin: